script/foreman-start-dev

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-./node_modules/.bin/webpack-dev-server --config config/webpack.config.js --host "::" --disable-host-check $WEBPACK_OPTS &
+./node_modules/.bin/webpack-dev-server-without-h2 --config config/webpack.config.js --host "::" --disable-host-check $WEBPACK_OPTS &
```

Comment:
Is `--disable-host-check` needed here if its specified `disableHostCheck: false` in the config?

Reply:
thanks, removed it